version: api.keploy.io/v1beta1
kind: SQL
name: mock-0
spec:
    metadata:
        operation: HANDSHAKE_RESPONSE
        responseOperation: MySQLErr
        type: config
    requests:
        - header:
            packet_length: 138
            packet_number: 1
            packet_type: HANDSHAKE_RESPONSE
          message:
            capability_flags: 12252111
            character_set: 224
            reserved: 23
            username: user
            auth_data: Qy9Ip9HivQt+pd5RLentxbkg8UA=
            database: mysql_db
            auth_plugin_name: mysql_native_password
            connect_attributes: {_client_name: Node-MySQL-2, _client_version: 3.7.1}
    responses:
        - header:
            packet_length: 74
            packet_type: MySQLHandshakeV10
          message:
            protocol_version: 10
            server_version: 5.7.44
            connection_id: 2
            auth_plugin_data: enIucWklIksyTkQTZgdJQwJ7HCsA
            capability_flags: 3254779903
            character_set: 8
            status_flags: 2
            auth_plugin_name: mysql_native_password
        - header:
            packet_length: 65
            packet_number: 2
            packet_type: MySQLErr
          message:
            error_code: 1044
            sql_state: "42000"
            error_message: Access denied for user 'user'@'%' to database 'mysql_db'
    created: 1705578109
